src/cur-proj.h

```diff
@@ -19,7 +19,7 @@
 #ifndef __CUR_PROJ_H__
 #define __CUR_PROJ_H__
 
-#include <proj.h>
+#include "proj.h"
```

Comment:
With the previous notation the code would not compile because the header is not being found.